morph/react/property-event-handler.js

```diff
@@ -14,9 +15,10 @@ export function enter(node, parent, state) {
   let importName = getImportNameForSource(node.source, state)
   // assuming it is a hook
   if (/^use[A-Z]/.test(node.value)) {
+    let expandedProps = getExpandedProps({ state, ignoreDefaultValues: true })
     let variableName = getVariableName(node.name, state)
     state.variables.push(
-      `let ${variableName} = ${importName}.${node.value}(VIEW_PROPS)`
+      `let ${variableName} = ${importName}.${node.value}(${expandedProps})`
```

Comment:
Cool, this should work. I don't remember if there was a reason why I used the expanded props at the end other than render only deals with the body of the component and not the function's signature but let's give this a go :)! Thanks!